File: solutions/python/darts/1/darts.py
import math

def score(x, y):
    pointRadius = math.sqrt(x*x + y*y)
    if pointRadius <= 1.0:
        return 10
    elif pointRadius <= 5.0:
        return 5
    elif pointRadius <= 10.0:
        return 1
    else:
        return 0
    

File: solutions/java/darts/1/src/main/java/Darts.java
class Darts {
    int score(int xDart, int yDart) {
        int x = Math.abs(xDart);
        int y = Math.abs(yDart);
        if (x <= 1 && y <= 1) return 10;
        if (x <= 3.5 && y <= 3.5) return 5;
        if (x <= 7 && y <= 7) return 1;
        return 0;
    }

    int score(double xDart, double yDart) {
        double x = Math.abs(xDart);
        double y = Math.abs(yDart);
        if (x <= 1 && y <= 1) return 10;
        if (x <= 3.5 && y <= 3.5) return 5;
        if (x <= 7 && y <= 7) return 1;
        
        return 0;
    }
}
